### PR TITLE
Adopt large block blob

### PR DESCRIPTION
**Pull request by troydai**
1. Maximize the upload block size and single put size.
2. Remove the mitigation for storage python SDK issue #190.
3. Update blob storage upload/download tests
4. Add live only tests for large blob download

### CODE REVIEW

**Comment by troydai** [comment]
Do not merge until after Monday.

File: src/azure-cli-testsdk/azure/cli/testsdk/base.py [hunk 1]
```diff
@@ -26,7 +26,76 @@
...
+    @classmethod
+    def set_env(cls, key, val):
+        os.environ[key] = val
+
+    @classmethod
+    def pop_env(cls, key):
+        return os.environ.pop(key, None)
+
+
+class ScenarioTest(LiveTest):  # pylint: disable=too-many-instance-attributes
```
Comment (tjprescott) [inline src/azure-cli-testsdk/azure/cli/testsdk/base.py]: A ScenarioTest isn't a live test, so this inheritance is a little bizarre. I would recommend having them both derive from something common (CliTest or something), rather than one inheriting from another. You could just as easily have LiveTest(ScenarioTest), which tells me something is fishy with the inheritance.

File: src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py [hunk 2]
```diff
@@ -131,7 +126,65 @@ def get_account_key(self, group, name):
                         .format(name, group)).output
 
 
-if __name__ == '__main__':
-    import unittest
+class StorageBlobUploadLiveTests(LiveTest):
+    @live_only()
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py]: Why is this necessary?  Shouldn't the fact that it's a LiveTest enforce the "live only" behavior?

Comment (troydai) [inline src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py]: Haven't find a way to let the test class itself to involve in test skipping. The `@skip` provided by `unittest` module is decorator based.
